src/components/GroupCards.js

```diff
@@ -0,0 +1,146 @@
+import React from 'react'
+import { graphql, StaticQuery, Link } from 'gatsby'
+import styled from 'styled-components'
+import clintonsquare from '../images/clintonsquare.jpg'
+
+const GroupContainer = styled.div`
+  height: 450px;
+  border: 1px solid #ddd;
+  border-radius: 4px;
+  box-shadow: 0 1px 1px rgba(0, 0, 0, 0.05);
+  display: flex;
+  flex-direction: column;
+
+  img {
+    height: 175px;
+    margin-bottom: 10px;
+    object-fit: cover;
+  }
+
+  header {
+    padding: 0 10px;
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 10px;
+  }
+
+  header b {
+    font-size: 18px;
+  }
+
+  .label {
+    padding: 0.2em 0.6em 0.3em;
+    color: white;
+    font-size: 75%;
+    border-radius: 0.25em;
+    height: 26px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .hackathon {
+    background: #d9534f;
+  }
+
+  .meetup {
+    background: #5bc0de;
+  }
+
+  p {
+    color: #777;
+    flex-grow: 1;
+    padding: 0 10px;
+    margin-bottom: 0;
+  }
+
+  .bottom {
+    background: #f5f5f5;
+    padding: 7px;
+    display: flex;
+    justify-content: flex-end;
+  }
+
+  .bottom a {
+    background: white;
+    border: 1px solid #ddd;
+    padding: 3px;
+    margin: 2px;
+    border-radius: 2px;
+    color: inherit;
+    text-decoration: none;
+    font-size: 75%;
+  }
+`
+
+const GroupGrid = styled.div`
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(300px, 1fr));
+  grid-gap: 30px;
+  margin-bottom: 20px;
+`
+
+const GroupCards = () => {
+  let query = graphql`
+    {
+      allMdx(filter: { fileAbsolutePath: { regex: "/src/pages/groups/" } }) {
+        edges {
+          node {
+            fileAbsolutePath
+            frontmatter {
+              title
+              group
+              summary
+              type
+            }
+          }
+        }
+      }
+    }
+  `
+
+  return (
+    <StaticQuery
+      query={query}
+      render={data => {
+        let groups = data.allMdx.edges.filter(
+          edge => edge.node.frontmatter.type !== null
```

Comment:
`type` is a bit generic as a term here.  Let's make it something more explicitly saying "this field existing means it will be displayed as a group card".

something like `type=group`, or `group=true`? then add something else to signify hackathon vs meetup, etc?